plugin/storage/cassandra/dependencystore/model.go

```diff
@@ -36,6 +37,8 @@ func (d *Dependency) MarshalUDT(name string, info gocql.TypeInfo) ([]byte, error
 		return gocql.Marshal(info, d.Child)
 	case "call_count":
 		return gocql.Marshal(info, d.CallCount)
+	case "source":
```

Comment:
what happens if you run this code against the schema where UDT was not upgraded? Will gocql simply not invoke this function for "source"?

Reply:
yes